pkg/machines/components/libvirtSlate.jsx

```diff
@@ -0,0 +1,76 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from "react";
+import cockpit from 'cockpit';
+import { mouseClick } from "../helpers.es6";
+import { startLibvirt } from "../actions.es6";
+
+const _ = cockpit.gettext;
+
+const LibvirtSlate = ({ libvirtState, dispatch }) => {
+    const goToLink = () => cockpit.jump("/system/services#/libvirtd.service");
+    const start = () => dispatch(startLibvirt());
+
+    let message;
+    let icon;
+    let detail;
+    let action;
+
+    switch (libvirtState.status) {
+        case 'active':
+            message = _("libvirt is active");
+            icon = (<span className="pficon-ok"/>);
+            break;
+        case 'unknown':
+            message = _("Connecting to libvirt");
+            icon = (<div className="spinner spinner-lg"/>);
+            break;
+        default:
+            message = _("libvirt is not active on the system");
+            icon = (<span className="fa fa-exclamation-circle"/>);
+            detail = (
+                <p>
+                    {_("See libvirt service for ")}<a onClick={mouseClick(goToLink)}>{_("more configuration.")}</a>
+                </p>
+            );
+            action = (
+                <div className="blank-slate-pf-main-action">
+                    <button className="btn btn-primary btn-lg" onClick={mouseClick(start)}>{_("Start libvirt")}</button>
```

Comment:
This button name should also be descriptive, similar to the header; that is, it should state: "Start Virtualization Service".

The docs link directly above explains that the "virtualization service" is libvirt.